### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by aslagle**
Updated a few dependencies and fixed warnings.

### CODE REVIEW

File: packages/opds-web-client/src/app.tsx [hunk 1]
```diff
@@ -23,12 +25,12 @@ class OPDSWebClient {
   }, elementId: string) {
     this.elementId = elementId;
     this.pathPattern = config.pathPattern || "/(collection/:collectionUrl/)(book/:bookUrl/)";
-    this.RouteHandler = React.createClass({
+    this.RouteHandler = createReactClass({
```
Comment (EdwinGuzman) [inline packages/opds-web-client/src/app.tsx]: Any plans on moving this soon to the `React.component` pattern?

Comment (aslagle) [reply packages/opds-web-client/src/app.tsx]: We can if you want :)

I think this is the only place it's used.

Comment (EdwinGuzman) [reply packages/opds-web-client/src/app.tsx]: Might be good to also remove the `react-create-class` dependency.

Comment (aslagle) [reply packages/opds-web-client/src/app.tsx]: Agreed.

**Pull request merged**